projects/batfish/src/org/batfish/grammar/cisco/Cisco_routemap.g4

```diff
@@ -43,19 +48,26 @@ boolean_rib_has_route_rp_stanza
    RIB_HAS_ROUTE IN rp_prefix_set
 ;
 
+boolean_rp_stanza
+:
+   boolean_and_rp_stanza
+   | boolean_rp_stanza OR boolean_and_rp_stanza
+;
+
 boolean_simple_rp_stanza
 :
    PAREN_LEFT boolean_rp_stanza PAREN_RIGHT
+   | boolean_as_path_in_rp_stanza
    | boolean_community_matches_any_rp_stanza
    | boolean_community_matches_every_rp_stanza
    | boolean_destination_rp_stanza
    | boolean_rib_has_route_rp_stanza
+   | boolean_tag_is_rp_stanza
 ;
 
-boolean_rp_stanza
```

Comment:
same as above. This specific diff is just an illusion - a single newline has moved.